### ISSUE TITLE
Auberdine to Rut'Theran Village Boat

### ISSUE THREAD

**Issue by TomCollinsLH** [LightsHope/server#1201]
I attempted to make my journey to Teldrassil as a Horde level 13 Hunter to tame an Owl as a pet. This requires you to die and corpse run over and over through Auberdine and Darnassus to make it to Teldrassil. I found when trying to run back to your corpse in Rut"Theran village the boat automatically resurrects you when you zone in when coming from Auberdine, essentially making it impossible for a low level Horde to get into Teldrassil to tame an Owl.

**Comment by brotalnia** [comment]
It is supposed to resurrect you if you die on a boat.

2.0.1 Patch Notes:
> Players will no longer resurrect with low mana and health if they died and released on a Zeppelin or Ship. 

http://eu.battle.net/wow/en/game/patch-notes/2-0-1

**Comment by TomCollinsLH** [comment]
I mean that is TBC, not Vanilla. And I did not die or release on the boat.

**Comment by brotalnia** [comment]
Patch 2.0.1 is the FIRST patch of TBC, and read what it actually says.

> Players will no longer resurrect

"No longer", meaning it resurrected you before, and what was before the first patch of TBC? Vanilla.

And it makes sense to be resurrected when you get on a boat or zeppelin, because usually you are changing maps. What are you going to do if you are dead, but your corpse is not even on the same map?

Here is more proof it resurrected you, ownedcore post from 2006.
> All you have to do is die to get into ghost form, then get on a boat. When the boat zones, youll have come back to lifeand your still on the boat!

https://www.ownedcore.com/forums/world-of-warcraft/world-of-warcraft-exploits/2428-ghost-traveling.html

**Comment by Cabro** [comment]
@brotalnia let me make it clear a bit. In TBC you were ressing with full HP+mana on boat zoning if you were dead. They just made it so you don't get ressed with LOW mana and hp anymore. Not that you will no longer resurrect, you should still resurrect.

**Comment by TomCollinsLH** [comment]
In Vanilla if a low level Horde wanted to run to Teldrassil they could by corpse running through Rut'Theran Village and Darnassus. How could this be the case if they res everytime on the boat? Keeping them from finishing the corpse runs. The Horde GY is in Darkshore, so when you die in Rut'Theran you must take the boat from Auberdine to retrieve your corpse.

**Comment by Cabro** [comment]
I don't understand one thing: how are you getting ressed? There is no zoning for that boat trip.

**Comment by TomCollinsLH** [comment]
Yes everytime I take the boat trip it causes a zoning loading screen which resses me. That must be the bug then, zoning when you're not supposed to. I had a GM watch me as it happened and he agreed there was definitely a bug.

**Comment by Cabro** [comment]
So the source or the bug is a custom change zoning on this boat trip?

**Comment by brotalnia** [comment]
Did any of you actually bother to read the 2006 thread i linked? He is describing this exact situation.

Ok, we all know the old method of ghost traveling was patched (Where you could die, then travel to places as a ghost and rez at the healer there)

> But heres a way to ghost travel, though not as good, still pretty nice. All you have to do is die to get into ghost form, then get on a boat. When the boat zones, youll have come back to lifeand your still on the boat!
> 
> Example, i used this to get to teldrassil and tame their low lvl owls. All i had to do, was take the boat there in ghost form. 

This is the same boat, so it is supposed to resurrect you.
